src/.env

```diff
@@ -0,0 +1,11 @@
+DEBUG=True
```

Comment:
This file should not be track, add this to the gitignore and also move to the mian dir ,instead you should add a .env.example with the vars values empty or a brief description to the repo